src/core/lv2/Lv2Proc.cpp

```diff
@@ -442,11 +442,16 @@ void Lv2Proc::initPlugin()
 
 	if (m_instance)
 	{
-		if(m_worker) {
+		const auto iface = static_cast<const LV2_Worker_Interface*>(
```

Comment:
Since you changed `setIface` to `setInterface`, I would think that `iface` and `m_iface` gets replaced with `interface` and `m_interface` as well.